apps/site/assets/ts/stop/components/amenities/ParkingAmenityCard.tsx

```diff
@@ -88,12 +106,34 @@ const getModalContent = (
                     </li>
                   )}
                 </ul>
+                {getPaymentElement(park)}
                 {park.capacity && (
                   <>
                     <h3>Facility Information</h3>
                     <ul>
                       <li>{park.capacity.total} total parking spots</li>
                       <li>{park.capacity.accessible} accessible spots</li>
+                      {park.manager && (
+                        <>
+                          {park.manager.name && (
+                            <li>Managed by {park.manager.name}</li>
+                          )}
+                          <ul>
+                            <li>
+                              Visit{" "}
+                              <a href={`${park.manager.url}`}>
```

Comment:
nit-pick(non-blocking): Should we check for the existence of the url and contact before displaying this?  Is it guaranteed that these fields are populated?  Same for the phone number.

Reply:
Only name and contact can be null. Will add check for contact and merge